addons/core/addon/helpers/relative-datetime-live.js

```diff
@@ -45,6 +45,10 @@ export default class extends Helper {
   // =methods
 
   compute([date]) {
+    if (!date) {
+      return;
+    }
```

Comment:
```suggestion
    if (!date) return;
```

Reply:
@randallmorey 
I have no strong opinion about 1 liner conditionals (I think is opinonated). 
But if this is what we want, I think could be benefitial for us as a team discuss it and enforced via prettier?

Reply:
I personally dislike them but I see that we've used them elsewhere so I've updated it for now. 

Totally agree on discussing it and enforcing it though

Reply:
I personally don't like them either, but since it's a personal opinion I think is fair to bring to FE tech time, discuss it as team, take a decision (which I will be totally fin with) and let the tooling enforcing it for us 😃